pkg/kratos/handlers.go

```diff
@@ -580,6 +603,18 @@ func NewAPI(service ServiceInterface, mfaEnabled bool, baseURL string, logger lo
 	return a
 }
 
+func setCookiesWithExclude(w http.ResponseWriter, cookies []*http.Cookie, exclude []string) {
```

Comment:
```suggestion
func setCookies(w http.ResponseWriter, cookies []*http.Cookie, exclude ...string) {
```

I'd say to override setCookies with a variadic so we keep it the same and should not break backwards compatibility

Reply:
ahh thanks. I knew there had to be a better way to do this, but I suck at go....